.github/workflows/main.yml

```diff
@@ -5,6 +5,11 @@ on:
   push:
```

Comment:
Please update the PR description to avoid closing the issue until there are test branches that make sure we did not regress anything.